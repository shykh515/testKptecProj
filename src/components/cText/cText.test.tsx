import React from 'react';
import { render } from '@testing-library/react-native';
import CText from './cText';

describe('Custom Test', () => {
  test('renders text correctly or not', () => {
    const { getByText } = render(<CText>Hello, Test!</CText>);
    expect(getByText('Hello, Test!')).toBeTruthy();
    expect(getByText).toMatchSnapshot();
  });

  // Add more tests for styling and other functionality if needed
});
