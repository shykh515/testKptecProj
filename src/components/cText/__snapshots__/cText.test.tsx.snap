// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Test renders text correctly or not 1`] = `[Function]`;
